packages/mdc-button/_mixins.scss

```diff
@@ -82,10 +84,6 @@
   }
```

Comment:
Could you help me add the inline comments about why we did the line-height in stroke?

It is here: https://github.com/material-components/material-components-web/pull/1305/files#diff-ea96fa70f2fa8b77e1de67b458429364